packages/edit-site/src/components/global-styles/font-sizes/font-sizes.js

```diff
@@ -117,9 +116,6 @@ function FontSizeGroup( {
 									</FlexItem>
 									<FlexItem>
 										<HStack justify="flex-end">
```

Comment:
Is this extra HStack still needed? Why not use the same pattern used for the shadows item?

Reply:
Hi @afercia, I've implemented the suggested changes in the latest commit. Thanks!